Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -149,6 +149,23 @@ final class PackageToolTests: XCTestCase {
         }
     }
 
+    func testShowDependencies_dotFormat_sr12016() throws {
+        // Confirm that SR-12016 is resolved.
+        // See https://bugs.swift.org/browse/SR-12016
+        fixture(name: "DependencyResolution/External/SR-12016") { prefix in
```

Comment:
Instead of a fixture on disk, can you try creating the package using test utilities? For example, see https://github.com/apple/swift-package-manager/blob/master/Tests/BuildTests/BuildPlanTests.swift#L102

Reply:
@aciidb0mb3r 

It's easy to use `func loadPackageGraph(fs:diagnostics:manifests:shouldCreateMultipleTestProducts:createREPLProduct) -> PackageGraph`, however, [`func dumpDependenciesOf(rootPackage:mode:)`](https://github.com/apple/swift-package-manager/blob/579125c5dbeb8635f8aa2a0ae00f467aa525d48b/Sources/Commands/show-dependencies.swift#L14) puts out the message directly to the standard output.

What way should we choose? (for testing)
* To use `dup2()`.
* To overhaul "show-dependencies.swift".
* Other way that I don't know.

Reply:
Ah. You can add a stream parameter with type `OutputByteStream` and emit the output on that stream. The default value can be stdoutStream and in tests you can pass in a BufferedOutputByteStream object.

Reply:
@aciidb0mb3r 
Thank you very much for the response, though you are busy.
I will go on trying to fix.